run/openssl2john.py

```diff
@@ -65,15 +65,16 @@ def process(filename, plaintext=None, cipher=0, md=0):
                 rdata = rdata.decode("ascii")
             sys.stdout.write("%s:$openssl$%s$%s$8$%s$%s$0$%s$%s$%s\n" %
                              (filename, cipher, md, salt, last_chunk,
-                              len(rdata), rdata, s))
+                              len(rdata) // 2, rdata, s))
```

Comment:
It's a fix for a bug I introduced recently when I was making openssl2john.py compatible with Python 3.x.